### DISCUSSION TITLE
Allow shared documents with nested documents to collapse the sidebar by default

### DISCUSSION THREAD

**Discussion by routerino** [outline/outline#5076] [Ideas]
This isn't a big enough feature request to warrant opening an issue, so I'll put it here and see what people think.

One thing I've noticed about moving my blog to outline is the [sidebar gets a bit unwieldy](https://blog.gurucomputing.com.au/s/blog). It would be nice if the default (or configurable) behavour allowed for the sidebar to be collapsed by default, stopping an information overload by having it all expanded.

**Comment by P-Godfroid** [comment]
I have exactly the same situation.
Having every little child page opened leads to too many information at once for the visitor.

And it seems another person too :
@kingfungdesign
https://github.com/outline/outline/discussions/4973

**Reply by Limezy** [reply]
Would + that one too !
Seems like an easy PR fix ? Probably out of my league saddly

**Reply by kingfungdesign** [reply]
Thanks for mentioning. Yes, I would love to have that option too!

**Comment by P-Godfroid** [comment]
My javascript for the web is unfortunately still a work in progress,
But I think these actions must be used

![image](https://user-images.githubusercontent.com/109596710/229047080-0b9feb81-fbff-471b-84f6-d63a20384aa3.png)
![image](https://user-images.githubusercontent.com/109596710/229047851-86a8864c-c2bf-44d9-af01-631c726655a5.png)

And the file where the changes must be made is this one :

https://github.com/outline/outline/blob/00baa2bd6d0e35bc8dcf663d1ab3c589df1a94bd/app/components/Sidebar/Sidebar.tsx

**Comment by mitchellcairns** [comment]
+1 for this. Would love to see a simple toggle whether a nested container is collapsed by default.
